### DISCUSSION TITLE
Electronic invoices

### DISCUSSION THREAD

**Discussion by gelbphoenix** [baimard/gestion#405] [Ideas]
Hey!

Would it be possible to add compatibility with electronic invoices?

*Explaination*

In 2014 the EU adopted the [Directive 2014/55/EU](http://data.europa.eu/eli/dir/2014/55) which forces it's member states to adopt standards for E-Invoicing for public procurement contracts. All public authorities are or will be forced to only recieve and process invoices complying with the coming standard. It shall be also applicable for companies in commercial transactions.

Seeing that it will become a standard for E-Invoicing and seeing that Gestion is mostly directed at small and medium companies, it should be discussed if it shall be implemented in this NC app.

(If that is already the plan or the category of this discussion is wrong I apologize)

**Comment by josefhajda** [comment]
also would be interested in this topic... as it will be mandatory to send e invoices with all information attached as xml it only makes sense to move to a new solution which supports this.... :) Is there a plan to implement this? Thanks!

**Comment by baimard** [comment]
Follow this discussion https://github.com/baimard/gestion/discussions/201
